src/main/webapp/components/results/result-index-cards.js

```diff
@@ -53,12 +35,16 @@ export default props => {
               <Actions disableSpacing={true}>
                 <Checkbox
```

Comment:
I would recommend making a separate custom checkbox component that uses the selection interface hook internally - that way you don't need to add selection to the React.memo dependency array